src/openforms/js/components/admin/forms/MappingArrayInput.js

```diff
@@ -0,0 +1,153 @@
+        <thead className="mapping-array-input">
+          <tr>
+            <th scope="col"></th>
+            <th scope="col">
+              <FormattedMessage
+                description="Label for 'key' in MappingArrayInput table column"
+                defaultMessage="Key"
+              />
+            </th>
+            <th scope="col">
+              <FormattedMessage
+                description="Label for 'value' in MappingArrayInput table column"
+                defaultMessage="Value"
+              />
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {copy.map(([key, value], index) => (
+            <tr key={index} className="mapping-array-input__item">
+              <td>
+                <div className="mapping-array-input__actions">
+                  <DeleteIcon
+                    onConfirm={onDelete.bind(null, index)}
+                    message={
+                      deleteConfirmationMessage ||
+                      intl.formatMessage({
+                        description:
+                          'Confirmation message to delete an item from a multi-value input',
+                        defaultMessage: 'Are you sure you want to delete this item?',
+                      })
+                    }
+                    icon="times"
+                  />
+                </div>
+              </td>
+              <td>
+                <input
+                  type={inputType}
+                  name="keyInput"
+                  value={key}
+                  onChange={onInputChange.bind(null, index)}
+                />
+              </td>
+              <td>
+                {valueArrayInput ? (
+                  <ArrayInput
+                    name="valueInput"
+                    key={key}
+                    values={value}
+                    onChange={onArrayInputChange.bind(null, index)}
+                    addButtonMessage={
+                      <span className="addlink">
+                        <FormattedMessage
+                          description="Label for add button used in MappingArrayInput"
+                          defaultMessage="Add another value"
+                        />
+                      </span>
+                    }
+                  />
+                ) : (
+                  <input
+                    type={inputType}
+                    name="valueInput"
+                    value={value}
+                    onChange={onInputChange.bind(null, index)}
+                  />
+                )}
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      <button type="button" className="button button--plain" onClick={onAdd}>
+        {addButtonMessage || (
+          <span className="addlink">
+            <FormattedMessage
+              description="Add item to multi-input field"
+              defaultMessage="Add item"
+            />
+          </span>
+        )}
```

Comment:
the <kbd>+</kbd> icon is missing when a message is provided:

![image](https://user-images.githubusercontent.com/5518550/223407398-77d75418-f1e8-4367-8704-c73337c9ac61.png)